test/inductor/test_torchinductor.py

```diff
@@ -554,6 +558,8 @@ def check_model_gpu(
         kwargs,
         atol=atol,
         rtol=rtol,
+        grad_atol=grad_atol,
+        grad_rtol=grad_rtol,
```

Comment:
There is another `check_model` call below on line 586.

Reply:
oof, missed that one when I split it off, thanks.